addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/parser/CommonRpcParser.java

```diff
@@ -67,7 +67,11 @@ protected Number toNumber(Object object) {
         if (object == null || object instanceof Number) {
             return (Number) object;
         }
-        return NumberUtils.createNumber(ObjectUtils.toString(object));
+        try {
+            return NumberUtils.createNumber(ObjectUtils.toString(object));
+        } catch (NumberFormatException ex) {
+            return null;
```

Comment:
Thanks for the PR! Imho some loggin is needed instead of silently swallowing this exception, wdyt?

Reply:
I'll create a PR to address this and the others :-)